src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo__link">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Company logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+              href="index.html"
+              class="nav__item is-active"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links start with the `#` symbol if they are used for page navigation. If these links are meant to lead to different pages, use proper URLs or paths.